mabwiser/linear.py

```diff
@@ -57,21 +58,51 @@ def predict(self, x):
 
         # Scale
         if self.scaler is not None:
-            x = x.reshape(1, -1)
-            x = self.scaler.transform(x.astype('float64')).reshape(-1)
+            x = self._scale_predict_context(x)
 
         # Calculate default expectation y = x * b
         return np.dot(x, self.beta)
 
+    def _scale_predict_context(self, x):
+        x = x.reshape(1, -1)
+        return self.scaler.transform(x.astype('float64')).reshape(-1)
+
+
+class _LinTS(_RidgeRegression):
+
+    def predict(self, x):
+
+        # Scale
+        if self.scaler is not None:
+            x = self._scale_predict_context(x)
+
+        # Randomly sample coefficients from Normal Distribution N(mean=beta, variance=exploration)
+        exploration = np.square(self.alpha) * self.A_inv
+
+        beta_sampled = self._multivariate_normal(exploration)
+
+        # Calculate expectation y = x * beta_sampled
+        return np.dot(x, beta_sampled)
+
+    def _multivariate_normal(self, exploration):
+        # Multivariate Normal Sampling
```

Comment:
can you add a quick word on what's exploration parameter? int float constant? array/matrix?